RELEASE.md

```diff
@@ -108,21 +115,15 @@ Signing a tag with a GPG key is appreciated, but in case you can't add a GPG key
 
 Once a tag is created, the release process through CircleCI will be triggered for this tag and Circle CI will draft the GitHub release using the `prombot` account.
 
-Now all you can do is to wait for tarballs to be uploaded to the Github release and the container images to be pushed to the Docker Hub and Quay.io. Once that has happened, click _Publish release_, which will make the release publicly visible and create a GitHub notification.
+Finally, wait for the build step for the tag to finish. The point here is to wait for tarballs to be uploaded to the Github release and the container images to be pushed to the Docker Hub and Quay.io. Once that has happened, click _Publish release_, which will make the release publicly visible and create a GitHub notification.
+** Note: ** for a release candidate version the _This is a pre-release_ box when drafting the release in the Github UI.
+
+### 3. Wrapping up
 
-### Wrapping up
+For release candidate versions, run the benchmark for 3 days using the `/benchmark x.y.z` command, `x.y.z` being the latest stable patch release of the previous minor release series.
```

Comment:
It has been changed to `/prombench` now, and I guess it's against a branch now and not a release? @krasi-georgiev / @geekodour can confirm.

Reply:
it's against a release tag, such as `/prombench v2.15.2`, I'll update this

Reply:
yep